registration/registration-client/src/main/java/io/mosip/registration/util/control/impl/HtmlFxControl.java

```diff
@@ -83,9 +83,10 @@ public FxControl build(UiSchemaDTO uiSchemaDTO) {
         getRegistrationDTo().getSelectedLanguagesByApplicant().forEach(langCode -> {
             final TitledPane titledPane = new TitledPane(resourceBundle.getString(langCode), buildWebView(langCode));
             accordion.getPanes().add(titledPane);
+titledPane.setId(uiSchemaDTO.getId()+langCode);
         });
 
-        accordion.setExpandedPane(accordion.getPanes().get(0));
+        //accordion.setExpandedPane(accordion.getPanes().get(0));
```

Comment:
Commented this line as scroll on the page not working with given id's to make checkbox approachable I made this change to run the script if I get any other alternate then will revert